benchmarks/distributed/rpc/rl/plot.py

```diff
@@ -0,0 +1,49 @@
+import matplotlib
```

Comment:
just wanna mention that this file does not belong to this repo. You can create your own repo to host plotting programs.